tests/basic/storage_pool_test.go

```diff
@@ -5794,3 +5794,226 @@ func addNewPools(n node.Node, numPools int) error {
+		log.FailOnError(Inst().V.EnterMaintenance(*nodeDetail), fmt.Sprintf("Failed to bring Pool [%s] to Mainteinance Mode on Node [%s]", poolUUID, nodeDetail.Name))
+
+		log.InfoD("Wait for Node to Enter Maintenance Mode")
+		WaitTillEnterMaintenanceMode(*nodeDetail)
+
+		// Delete the Pool with Invalid Pool ID
+		err = Inst().V.DeletePool(*nodeDetail, invalidPoolID)
+		dash.VerifyFatal(err != nil, true, fmt.Sprintf("Pool Delete Status is SUCCESS. Expected Failure! : Node Detail [%v]", nodeDetail))
+		log.InfoD("Deleting Pool with InvalidID Errored as expected [%v]", err)
+
+		log.InfoD("Bring Node out of Maintenance Mode")
+		ExitFromMaintenanceMode(*nodeDetail)
+
+		JustAfterEach(func() {
+			defer EndTorpedoTest()
+			log.InfoD("Exit from Maintenance mode if Pool is still in Maintenance")
+			ExitNodesFromMaintenanceMode()
+			AfterEachTest(contexts, testrailID, runID)
+		})
+	})
+})
+
+var _ = Describe("{PoolResizeInvalidPoolID}", func() {
+	var testrailID = 79487
+	// Testrail Description : Resize with invalid pool ID
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/84470
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolResizeInvalidPoolID",
+			"Resize with invalid pool ID",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Resize with invalid pool ID"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("deleteinvalidpoolid-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
+
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+
+		// invalidPoolUUID Generation
+		id := uuid.New()
+		invalidPoolUUID := fmt.Sprintf("%s", id.String())
+
+		// Resize Pool with Invalid Pool ID
+		// Do Pool Expand on the Node
+		stepLog = fmt.Sprintf("Expanding pool on node [%s] and pool UUID: [%s] using auto", nodeDetail.Name, poolUUID)
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			poolToBeResized, err := GetStoragePoolByUUID(poolUUID)
+			log.FailOnError(err, "Failed to get pool using UUID [%s]", poolUUID)
+			expectedSize := poolToBeResized.TotalSize * 2 / units.GiB
+
+			log.InfoD("Current Size of the pool [%s] is [%d]", poolUUID, poolToBeResized.TotalSize/units.GiB)
+
+			// Now trying to Expand Pool with Invalid Pool UUID
+			err = Inst().V.ExpandPool(invalidPoolUUID, api.SdkStoragePool_RESIZE_TYPE_AUTO, expectedSize)
+			fmt.Errorf("%v", err)
+			// dash.VerifyFatal(err, nil, "Pool expansion init successful?")
```

Comment:
Handle this error properly and remove commented out code

Reply:
My Bad !! I will un comment the line , and remove fmt.Errorf(.*)